walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/handlers/OpenId4VCIRequestHandler.kt

```diff
@@ -115,7 +116,14 @@ internal class OpenId4VCIRequestHandler(
             credentialMetadata.transformLocalizedIssuerDisplayDefinitionToRequesterStyle()
         val verifiedIdStyle =
             credentialConfiguration.getVerifiedIdStyleInPreferredLocale(requesterStyle.name)
-        val requirement = transformToRequirement(credentialConfiguration.scope, credentialOffer)
+        val accessTokenEndpoint = fetchAccessTokenEndpointFromOpenIdWellKnownConfig(
```

Comment:
should this throw an error if the credentialMetadata.credential_issuer is nil?

Reply:
validation happens on line 46 before this method is invoked but error can be thrown here as well so that this method is self sufficient.